### ISSUE TITLE
Unit Test? 

### ISSUE THREAD

**Issue by siolag161** [asaglimbeni/django-datetime-widget#71]
I was working on #33 and then realized you don't have any unit test, yet. Don't you think it's high time we added some?

**Comment by ollytheninja** [comment]
Yes, probably one of the reasons I didn't get further with #33 myself.
